jasmine/defs.bzl

```diff
@@ -1,16 +1,9 @@
 "Public API re-exports"
 
 load("@aspect_bazel_lib//lib:copy_file.bzl", "copy_file")
-load("@aspect_rules_jasmine//jasmine/private:jasmine_test.bzl", "lib")
-load("@aspect_rules_js//js:libs.bzl", "js_binary_lib")
+load("@aspect_rules_jasmine//jasmine/private:jasmine_test.bzl", _jasmine_test = "jasmine_test")
 
-_jasmine_test = rule(
```

Comment:
Was there a docs reason for moving this?

Reply:
It fixes #56 - no that could possibly be in a separate PR from the docs rename.